frontend/javascripts/libs/window.ts

```diff
@@ -72,6 +74,7 @@ const _window =
         addEventListener,
         removeEventListener,
         open: (_url: string) => {},
+        performance: { now: () => ++performanceCounterForMocking },
```

Comment:
`performance.now()` needs to be mocked for the tests. By increasing a counter I want to simulate "time moving on" and therefore avoid that a calculation like `roundTripTime = endTime - startTime` results in a value of 0. A value of 0 might lead to a `division by 0` error and that's what I want to avoid.